### DISCUSSION TITLE
Move of downloaded deezer file not working

### DISCUSSION THREAD

**Discussion by lelus78** [RandomNinjaAtk/docker-lidarr-extended#149] [Q&A] [answered]
Hello, I'm new to doker image, so it's highly probable that I have made some mistake on the configuration,

Te problem present like this, I can see the file getting downloaded in the log file, if I monitor the directory I will see the file been created in \downloads-lidarr-extended\incomplete, but after a few second the track downloaded are not moved to /music  and they disappear from all my drive.

[album_8082924_9] Leonard Cohen - Ain't No Cure for Love :: Downloading track. Downloading 11579558 bytes.
[album_8082924_9] Download at 14%
[album_8082924_9] Download at 16%
[album_8082924_9] Download at 18%
[album_8082924_9] Download at 20%
[album_8082924_9] Download at 22%
[album_8082924_9] Download at 24%
[album_8082924_9] Leonard Cohen - Ain't No Cure for Love :: Track downloaded.
[album_8082924_9] Leonard Cohen - Ain't No Cure for Love :: Tagging track.
[album_8082924_9] Leonard Cohen - Ain't No Cure for Love :: Track tagged.
[album_8082924_9] Completed download of /102 - Ain't No Cure for Love.mp3

Let me know if need more information.

**Comment by RandomNinjaAtk** [comment]
You need to provide a bit more log data than that because the log will indicate what is happening and you need provide your docker configuration/parameter settings.

**Comment by lelus78** [comment]
Ok I will add my Audio.log and provide the configuration. let me know if need other info.

[Audio.txt](https://github.com/RandomNinjaAtk/docker-lidarr-extended/files/10154105/Audio.txt)

version: "2.1"
services:
  lidarr-extended:
    image: randomninjaatk/lidarr-extended:latest
    container_name: lidarr-extended
    volumes:
      - E:\Docker image\Lidarr\Config:/config
      - E:\Docker image\Lidarr\downloads-lidarr-extended:/downloads-lidarr-extended
      - M:\Organizzata:/music
      - E:\Docker image\Lidarr\music-videos:/music-videos
    environment:
      - TZ=America/New_York
      - enableAudioScript=true
      - enableVideoScript=false
      - scriptInterval=15m
      - configureLidarrWithOptimalSettings=true
      - searchSort=date
      - audioFormat=native
      - audioBitrate=lossless
      - requireQuality=true
      - enableReplaygainTags=true
      - audioLyricType=both
      - dlClientSource=deezer
      - arlToken=a381ca62db360de3c1993ad39231a9caa71804142528b1f69da2e2a61298d353da89a6c76dcf2701c7c64b99aaf3118fe3bbb76a54b15bcc3a9d7976b3689293d730096fafb5dca4bc929b22dca5cd88c21ea78b6e2d0d2649ca29832118bc7c
      - tidalCountryCode=IT
      - addDeezerTopArtists=true
      - addDeezerTopAlbumArtists=true
      - addDeezerTopTrackArtists=true
      - topLimit=10
      - addRelatedArtists=false
      - numberOfRelatedArtistsToAddPerArtist=5
      - lidarrSearchForMissing=true
      - addFeaturedVideoArtists=false
      - plexUrl=http://192.168.1.50:32400
      - plexToken=2mH6Ms2kAmjMSpBxC2Eq
      - youtubeSubtitleLanguage=en
      - enableQueueCleaner=true
      - matchDistance=5
      - enableBeetsTagging=true
      - beetsMatchPercentage=90
      - downloadPath=/downloads-lidarr-extended
      - videoPath=/music-videos
      - SMA_PATH=/usr/local/sma
    ports:
      - 8686:8686
    restart: unless-stopped

**Comment by RandomNinjaAtk** [comment, marked as answer]
Without reading your log, but based on your original log and posted config. The issue is caused by your config... https://github.com/RandomNinjaAtk/docker-lidarr-extended#parameters

What do you think "- requireQuality=true" does?

Also hint from your log file... 
```
2022-12-04 19:04:22 :: Audio :: 1.0.300 :: 1 :: missing :: 1 of 68 :: Kelly Clarkson :: Since U Been Gone (Juliet’s version) :: Single :: Checking for unwanted files
2022-12-04 19:04:22 :: Audio :: 1.0.300 :: 1 :: missing :: 1 of 68 :: Kelly Clarkson :: Since U Been Gone (Juliet’s version) :: Single :: Unwanted files found!
2022-12-04 19:04:22 :: Audio :: 1.0.300 :: 1 :: missing :: 1 of 68 :: Kelly Clarkson :: Since U Been Gone (Juliet’s version) :: Single :: Performing cleanup...
```

**Comment by lelus78** [comment]
MMM ok, I misunderstood, does that mean in my case, it requires lossless format correct? if is not the file are deleted.  So i can change the requirement quality to false to keep my mp3-320 or specify mp3 in the audioFormat.

**Reply by RandomNinjaAtk** [reply]
Yarp

**Reply by lelus78** [reply]
Thanks for your help.
